--- C:\DEV\workspace\REZD-new\src\kytten\dialog.py	2009-11-20 09:39:57.000000000 -0800
+++ dialog.py	2010-10-04 17:48:49.000000000 -0700
@@ -174,8 +174,9 @@
         if self.focus is not None:
             self.focus.dispatch_event('on_mouse_release',
                                       x, y, button, modifiers)
-        DialogEventManager.on_mouse_motion(self, x, y, 0, 0)
-        return pyglet.event.EVENT_HANDLED
+            DialogEventManager.on_mouse_motion(self, x, y, 0, 0)
+            return pyglet.event.EVENT_HANDLED
+        return pyglet.event.EVENT_UNHANDLED
 
     def on_mouse_scroll(self, x, y, scroll_x, scroll_y):
         """
@@ -640,4 +641,4 @@
             ])),
             window=window, batch=batch, group=group,
             theme=theme, movable=True,
-            on_enter=on_ok_click, on_escape=on_cancel_click)
\ No newline at end of file
+            on_enter=on_ok_click, on_escape=on_cancel_click)
